test/e2e/storage/persistent_volumes-disruptive.go

```diff
@@ -108,6 +109,107 @@ var _ = SIGDescribe("PersistentVolumes[Disruptive][Flaky]", func() {
 		framework.DeletePodWithWait(f, c, nfsServerPod)
 	})
 
+	Context("when kube-controller-manager restarts", func() {
+		var (
+			diskName1, diskName2 string
+			err                  error
+			pvConfig1, pvConfig2 framework.PersistentVolumeConfig
+			pv1, pv2             *v1.PersistentVolume
+			pvc1, pvc2           *v1.PersistentVolumeClaim
+			clientPod            *v1.Pod
+		)
+
+		BeforeEach(func() {
+			framework.SkipUnlessProviderIs("gce", "gke")
+			framework.SkipUnlessSSHKeyPresent()
+
+			By("Initializing first PD with PVPVC binding")
+			diskName1, err = framework.CreatePDWithRetry()
```

Comment:
Under test/upgrades/storage/persistent_volumes.go, there's actually a createGCEVolume() method.  Maybe you can pull that out into volume_util.go and reuse it here.